client/my-sites/add-ons/main.tsx

```diff
@@ -62,36 +60,16 @@ const ContainerMain = styled.div`
 
 const ContentWithHeader = ( props: { children: ReactElement } ) => {
 	const translate = useTranslate();
-	const isWide = useDesktopBreakpoint();
-	const selectedSite = useSelector( getSelectedSite );
-
-	const navigationItems = [
-		{
-			label: translate( 'Add-Ons' ) as string,
-			href: `/add-ons/${ selectedSite?.slug }`,
-			helpBubble: (
-				<span>
-					{ translate(
-						'Expand the functionality of your WordPress.com site by enabling any of the following features.'
-					) }
-				</span>
-			),
-		},
-	];
 
 	return (
 		<ContainerMain>
 			<Main className="add-ons__main" wideLayout>
-				<FixedNavigationHeader compactBreadcrumb={ ! isWide } navigationItems={ navigationItems } />
 				<DocumentHead title={ translate( 'Add-Ons' ) } />
-				<FormattedHeader
-					className="add-ons__formatted-header"
-					brandFont
-					headerText={ translate( 'Boost your plan with add-ons' ) }
-					subHeaderText={ translate(
+				<NavigationHeader
+					title={ translate( 'Boost your plan with add-ons' ) }
+					subtitle={ translate(
```

Comment:
Re @roo2's point here https://github.com/Automattic/wp-calypso/pull/83328#issuecomment-1776222993

So we want to keep the branded heading but add a new NavigationHeader for "Add-Ons" instead?

Reply:
I considered it, but the headers seem redundant since the info icon is just the subheading below, and "Add-ons" is in both headers. Including both doesn't seem beneficial to me and breaks with our unification effort.
<img width="1073" alt="Screenshot 2023-10-24 at 2 08 39 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/b3e1a1de-5789-4fc8-b7eb-9bac4e5c93f7">